### DISCUSSION TITLE
NanoEls H2 build with dividing attachement

### DISCUSSION THREAD

**Discussion by fmw626** [kachurovskiy/nanoels#118] [Show and tell]
Hello everyone,

I wanted to share my NanoEls H2 Build, which I assembled a few months ago and have been using extensively.

Here are some noteworthy details about my build:

- Spindle Encoder Placement: Taking inspiration from [EdFleta's build](https://github.com/kachurovskiy/nanoels/discussions/87), I decided to conceal the spindle encoder within the hollow lathe head.

- Increased Torque: To address motor stalling issues, I built the largest pulleys possible. By maximizing the torque, I have achieved smoother and more reliable performance during operation.

- Dividing Attachment: As an additional feature, I built a dividing attachment for the lathe. It incorporates a fairly large 3D-printed worm wheel and a worm positioned on a shaft, which can be easily engaged or disengaged with a single nut. Currently, it can only be operated manually. [However, my ultimate goal is to integrate this dividing function into the NanoEls H4.](https://github.com/kachurovskiy/nanoels/discussions/119)

Thank you to Maxim and the community for their fantastic work that inspired me in this project!

fmw

![IMG_7174](https://github.com/kachurovskiy/nanoels/assets/122809499/990ed769-1fce-411d-bfaa-b4b7f1c6f968)
![IMG_7175](https://github.com/kachurovskiy/nanoels/assets/122809499/da085f10-a459-40d5-8184-5d566582a28d) ![IMG_7177](https://github.com/kachurovskiy/nanoels/assets/122809499/89e44166-8c8f-4cea-9718-fdf7dd4fed5d) ![IMG_7178](https://github.com/kachurovskiy/nanoels/assets/122809499/75c8de60-f062-483a-8aeb-8f390ea4bfc6)
![IMG_7182](https://github.com/kachurovskiy/nanoels/assets/122809499/9efc1173-d15d-4854-b7c8-3160caabe725)
![IMG_7183](https://github.com/kachurovskiy/nanoels/assets/122809499/424c77ae-d653-4d63-aedc-2fed1c1ed431) ![IMG_7184](https://github.com/kachurovskiy/nanoels/assets/122809499/0277073e-889a-43d5-89c5-f236bb61ca2e)

**Comment by kachurovskiy** [comment]
Hi @fmw626, thank you so much for sharing your build, it's awesome! Very clean and such an interesting way of doing a dividing head. Well done
